parts/k8s/addons/kubernetesmasteraddons-kube-proxy-daemonset.yaml

```diff
@@ -20,6 +20,10 @@ spec:
         operator: Equal
         value: "true"
         effect: NoSchedule
+      - operator: "Exists"
+        effect: NoExecute
+      - operator: "Exists"
+        effect: NoSchedule
       containers:
```

Comment:
This is to ensure that we never schedule kube-proxy on master, correct?

Reply:
It's the opposite: it ensures that kube-proxy will always be able to be scheduled on *any* node, regardless of taint.